### PR TITLE
chore: update example pipelines documentation

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
chore: update example pipelines documentation

this PR updates documentation for example pipelines. It corrects links to SSP operator and update pipeline tasks
Signed-off-by: Karel Šimon <ksimon@redhat.com>

**Release note**:
```
Update example pipelines documentation

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Virt-sysprep can be used for preparing VM images which can be then used as base
 This example shows how you can update an operating system and seal VM's image by using virt-customize.
 Then, a VM is created from such image.
 
-#### [Windows 10 Installer Pipeline](examples/pipelines/windows10-installer)
+#### [Windows bios Installer Pipeline](examples/pipelines/windows-bios-installer)
```
Comment (0xFelix) [inline README.md:105]: `bios` -> `BIOS` to make it look nicer?

File: examples/pipelines/windows-bios-installer/README.md [hunk 2]
```diff
@@ -8,20 +8,15 @@ installation of Windows is automatically executed and controlled by a Windows an
...
+- KubeVirt `v1.0.0`
+- Tekton Pipelines `v0.44.0`
 
 ## Links
 
-- [Windows 10 Installer Pipeline for Kubernetes](https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/kubernetes/windows-bios-installer-pipeline.yaml)
-- [Windows 10 Installer PipelineRun for Kubernetes](windows10-installer-pipelinerun-kubernetes.yaml)
-- [Windows 10 Installer Pipeline for OKD](https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/okd/windows-bios-installer-pipeline.yaml)
-- [Windows 10 Installer PipelineRun for OKD](windows10-installer-pipelinerun-okd.yaml)
+- [Windows 10 Installer Pipeline](https://github.com/kubevirt/ssp-operator/blob/main/data/tekton-pipelines/windows-bios-installer-pipeline.yaml)
```
Comment (0xFelix) [inline examples/pipelines/windows-bios-installer/README.md:18]: BIOS Installer Pipeline?

File: examples/pipelines/windows-efi-installer/README.md [hunk 3]
```diff
@@ -19,19 +19,16 @@ automatically executed and controlled by a Windows answer file. Then the pipelin
...
-- KubeVirt `v0.59.1`
+- KubeVirt `v1.0.0`
 - Tekton Pipelines `v0.44.0`
 
 ## Links
 
-- [Windows efi Pipeline for okd](https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/okd/windows-efi-installer-pipeline.yaml)
-- [Windows 11 efi Installer PipelineRun for okd](windows11-installer-pipelinerun-okd.yaml)
-- [Windows 2k22 efi Installer PipelineRun for okd](windows2k22-installer-pipelinerun-okd.yaml)
+- [Windows efi Pipeline](https://github.com/kubevirt/ssp-operator/blob/main/data/tekton-pipelines/windows-efi-installer-pipeline.yaml)
```
Comment (0xFelix) [inline examples/pipelines/windows-efi-installer/README.md:29]: `efi` -> `EFI` to make it look nicer?

File: examples/pipelines/windows-efi-installer/windows11-installer-pipelinerun.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: windows11-installer-run-
+spec:
+  params:
+    - name: winImageDownloadURL
+      value: >-
+       <INSERT_WINDOWS_ISO_URL>
```
Comment (0xFelix) [inline examples/pipelines/windows-efi-installer/windows11-installer-pipelinerun.yaml:9]: Multiline string needed? I would keep it in sync with `windows10-installer-pipelinerun.yaml`.

File: examples/pipelines/windows-efi-installer/windows2k22-installer-pipelinerun.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: windows2k22-installer-run-
+spec:
+  params:
+    - name: winImageDownloadURL
+      value: >-
```
Comment (0xFelix) [inline examples/pipelines/windows-efi-installer/windows2k22-installer-pipelinerun.yaml:8]: Same.

File: examples/pipelines/windows-efi-installer/README.md (hunk 3 above)
Comment (ksimon1) [reply examples/pipelines/windows-efi-installer/README.md]: fixed

**Review by 0xFelix — APPROVED** [review]
Thanks!

/lgtm

**Pull request merged**